docs/content/preview/migrate/airgapped.md

```diff
@@ -0,0 +1,48 @@
+<!--
++++
+private=true
++++
+-->
+
+You can perform an airgapped installation on RHEL 7/8, CentOS 7/8, and Ubuntu OS.
+
+For airgapped installations, you need to download all the rpm files that are necessary for installing and running `yb-voyager` on a machine with an internet connection.
```

Comment:
Please modify this line to:
`For airgapped installations, you need to download the tarball containing all the rpm files that are necessary for installing and running `yb-voyager` on a machine with an internet connection. You must download the tarball corresponding to your airgapped machine's OS. You can then transfer these files to your airgapped machine and proceed with the installation using the steps below.`